selinon/system_state.py

```diff
@@ -162,6 +164,7 @@ def __init__(self, dispatcher_id, flow_name, node_args=None, retry=None, state=N
         # Instantiate lazily later if we will know that there is something to process
         self._waiting_edges = []
         self._retry = retry
+        self._config = copy.copy(config)
```

Comment:
Why there is needed a copy?

Reply:
Use case when parallel execution a different reference was needed hence use the copy.

Reply:
Was it process level parallel execution?